amespahdbpythonsuite/amespahdb.py

```diff
@@ -0,0 +1,707 @@
+        """
+        Retrieve and return transitions object based on UIDs input.
+        UIDs can be a list, e.g. the output of search method, or an integer.
+        Calls the :meth:`amespahdbpythonsuite.transitions.transitions` class.
+
+        Parameters
+        ----------
+        uids : list of integers
+            List of UIDs.
+
+        Returns
+        -------
+        transitions object
+
+        """
+        from amespahdbpythonsuite.transitions import transitions
+        if type(uids) == int:
+            uids = [uids]
+
+        return transitions(type=self.__data['database'],
+                           version=self.__data['version'],
+                           data=self.__getkeybyuids('transitions', uids),
+                           pahdb=self.__data,
+                           # pahdb=self.subdatabase(uids),
+                           uids=uids,
+                           model={'type': 'zerokelvin_m',
+                                  'temperature': 0.0,
+                                  'description': ''},
+                           units={'abscissa': {'unit': 1,
+                                               'str': 'frequency [wavenumber]'},
+                                  'ordinate': {'unit': 2,
+                                               'str': 'integrated cross-section [km/mol]'}}
+                           )
+
+    def getlaboratorybyuid(self, uids):
+        """
+        Retrieve and return laboratory database object based on UIDs input.
+        UIDs can be a list, e.g. the output of search method, or an integer.
+        Calls the :meth:`amespahdbpythonsuite.laboratory.laboratory` class.
+
+        Parameters
+        ----------
+        uids : list of integers
+            List of UIDs.
+
+        Returns
+        -------
+        laboratory database object
+
+        """
+        # Check if the experimental database is loaded.
+        if self.__data['type'] != 'experimental':
+
+            self.message('EXPERIMENTAL DATABASE REQUIRED')
+
+            return None
+
+        from amespahdbpythonsuite.laboratory import laboratory
+
+        if type(uids) == int:
+            uids = [uids]
+
+        return laboratory(type=self.__data['type'],
+                          version=self.__data['version'],
+                          data=self.__getkeybyuids('laboratory', uids),
+                          pahdb=self.__data,
+                          uids=uids,
+                          model={'type': 'laboratory_m',
+                                 'temperature': 0.0,
+                                 'description': ''},
+                          units={'abscissa': {'unit': 1,
+                                              'str': 'frequency [wavenumber]'},
+                                 'ordinate': {'unit': 2,
+                                              'str': 'absorbance [-log(I/I$_{0})$'}}
+                          )
+
+    def getspeciesbyuid(self, uids):
```

Comment:
It is probably best to implement a 'mock' species-class so that the Travis build doesn't fail.